### DISCUSSION TITLE
更新后加了随机机制，注意细节

### DISCUSSION THREAD

**Discussion by MlgmXyysd** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#5] [Show and tell]
例如JVM这道题，不同用户看到的选项可能是不一样的
G1垃圾回收器将堆内存划分为多个大小**不等**的Region
G1垃圾回收器将堆内存划分为多个大小**相等**的Region
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/60f2f54b-06f0-4317-87ee-2c8739392eec)
MTK刷机题目也是
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/c72f7e8d-fae6-4e06-9fee-70c982639d89)
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/9c488ca1-d7ec-487d-b5de-1b3822a51f84)

**Comment by Jonah-77** [comment]
骂骂咧咧
